target/linux/ipq806x/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -9,6 +9,9 @@ board=$(board_name)
 case "$FIRMWARE" in
 "ath10k/pre-cal-pci-0000:01:00.0.bin")
 	case $board in
+	askey,rt4230w-rev6)
+		caldata_extract "art" 0x1000 0x3f1f
```

Comment:
Your extract size is different from the other devices. Are you sure you need that much?

Reply:
I thought I needed to define the start and end of each one

Reply:
It's start and size.